packages/flutter/lib/src/material/curves.dart

```diff
@@ -0,0 +1,36 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/animation.dart';
+
+// The easing curves of the Material Library
+
+/// The standard easing curve in the Material specification.
+///
+/// Elements that begin and end at rest use standard easing.
+/// They speed up quickly and slow down gradually, in order
+/// to emphasize the end of the transition.
+///
+/// See also:
+/// * <https://material.io/design/motion/speed.html#easing>
+const Curve standardEasing = Cubic(0.4, 0.0, 0.2, 1);
```

Comment:
This is the same as the existing fastOutSlowIn curve. If this name is preferable, a separate PR could migrate the SDK the new name and deprecate the old one.

Reply:
One name is not preferred over the other. I'll mention that it's the same as fastOutSlowIn, which is what Android folks are used to

Reply:
If we're going to hang on to both names, then it would be a little clearer if this was:
```dart
const Curve standardEasing = fastOutSlowIn;
```